src/Magento/FunctionalTestingFramework/Console/DoctorCommand.php

```diff
@@ -128,9 +129,17 @@ private function checkAuthenticationToMagentoAdmin()
             $this->ioStyle->success('Successful');
             $result = true;
         } catch (TestFrameworkException $e) {
+            if (getenv('MAGENTO_BACKEND_BASE_URL')) {
+                $urlVar = 'MAGENTO_BACKEND_BASE_URL';
+            } else {
+                $urlVar = 'MAGENTO_BASE_URL';
+            }
             $this->ioStyle->error(
-                $e->getMessage()
-                . "\nPlease verify MAGENTO_ADMIN_USERNAME and MAGENTO_ADMIN_PASSWORD in .env."
+                $e->getMessage() . "\n"
+                . "Please verify the following configuration settings in .env:\n"
```

Comment:
to avoid confusion with the display of invalid values, maybe just have this -
`            $this->ioStyle->error(
                $e->getMessage()
                . "\nPlease verify if " .$urlVar . ", " . "MAGENTO_ADMIN_USERNAME and MAGENTO_ADMIN_PASSWORD in .env are valid."
            );
`

Reply:
Changed.